offchain-modules/src/scripts/integration-test/eth.ts

```diff
@@ -107,15 +133,65 @@ async function main() {
     const ckbMintRecord = ckbMintRecords[0];
     assert(ckbMintRecord.chain === ChainType.ETH);
     assert(ethLockRecord.sudtExtraData === sudtExtraData);
-    assert(ckbMintRecord.status === 'todo');
+    assert(ckbMintRecord.status === 'success');
     assert(ckbMintRecord.asset === ETH_ADDRESS);
     assert(ckbMintRecord.amount === amount.toHexString());
-    assert(ckbMintRecord.recipientLockscript === recipientLockscript);
+    assert(ckbMintRecord.recipientLockscript === `0x${toHexString(recipientLockscript)}`);
+
+    // check sudt balance.
+    const account = new Account(PRI_KEY);
+    const ownLockHash = ckb.utils.scriptToHash(<CKBComponents.Script>await account.getLockscript());
+    const asset = new EthAsset('0x0000000000000000000000000000000000000000', ownLockHash);
+    const bridgeCellLockscript = {
+      codeHash: ForceBridgeCore.config.ckb.deps.bridgeLock.script.codeHash,
+      hashType: ForceBridgeCore.config.ckb.deps.bridgeLock.script.hashType,
+      args: asset.toBridgeLockscriptArgs(),
+    };
+    const sudtArgs = ckb.utils.scriptToHash(<CKBComponents.Script>bridgeCellLockscript);
+    const sudtType = {
+      codeHash: ForceBridgeCore.config.ckb.deps.sudtType.script.codeHash,
+      hashType: ForceBridgeCore.config.ckb.deps.sudtType.script.hashType,
+      args: sudtArgs,
+    };
+    const balance = await collector.getSUDTBalance(
+      new Script(sudtType.codeHash, sudtType.args, sudtType.hashType),
+      await account.getLockscript(),
+    );
+
+    if (!sendBurn) {
+      logger.debug('sudt balance:', balance.toHexString());
+      logger.debug('expect balance:', Amount.fromUInt128LE(amount.toHexString()).toHexString());
+      assert(balance.eq(Amount.fromUInt128LE(amount.toHexString())));
```

Comment:
balance should compare with new Amount(amount),  because of amount.toHexString() is NOT Uint128LE